features/step_definitions/company_steps.rb

```diff
@@ -88,11 +92,13 @@
 
 And "I hide the companies search form" do
   step %{I click on t("accordion_label.company_search_form.hide")}
+  step %{I wait for all ajax requests to complete}
```

Comment:
Why is this statement added?

Reply:
Search is now submitted as an Ajax request.  I was finding that some test dynamics weren't working because the test would look for (or confirm not present) something in the search results _before_ the results table was updated.

With the change to " .... in the list of companies" for those test steps I don't need this anymore.